docs/integrations/avid/index.md

```diff
@@ -0,0 +1,66 @@
+# 📒 AVID
+
+```{toctree}
+:caption: Table of Contents
+:maxdepth: 1
+:hidden:
+
+./avid-integration-llm.ipynb
+```
+
+## What is AVID?
```

Comment:
nitpick: if you want to stick to the same h2s as in the other integrations, it would be `Why AVID` instead.